cogs/hostbot.py

```diff
@@ -794,6 +800,273 @@ async def addspec_on(self, ctx: commands.Context):
 
         await ctx.message.add_reaction(ctx.bot.greentick)
 
+    @commands.command()
+    @commands.has_permissions(administrator=True)
+    @commands.guild_only()
+    async def resolve(self, ctx: commands.Context, *, night: int):
+        big_sheet_name = 'Formido Oppugnatura Exsequens'
```

Comment:
This and other hardcoded cases have a commented section to replace with (after the current EIMM game is over)